velox/exec/Task.h

```diff
@@ -83,6 +93,13 @@ class Task : public std::enable_shared_from_this<Task> {
   // in debugging messages and listings.
   static std::string shortId(const std::string& id);
 
+  // Register user-defined join bridge implementation for user-define join
+  // operator
+  static void registerJoinBridgeTranslator(
```

Comment:
These methods won't be needed if we re-use PlanNodeTranslator.

Reply:
Fixed